### PR TITLE
OPENEUROPA-2264: Add sticky flag for globan.

### PR DESCRIPTION
**Pull request by nagyad**

### CODE REVIEW

File: modules/oe_webtools_globan/src/Form/WebtoolsGlobanSettingsForm.php [hunk 1]
```diff
@@ -106,6 +106,17 @@ public function buildForm(array $form, FormStateInterface $form_state): array {
...
     ];
 
+    $form['globan_settings']['sticky'] = [
+      '#type' => 'select',
+      '#title' => $this->t('Sticky'),
+      '#options' => [
+        0 => $this->t('No'),
+        1 => $this->t('Yes'),
+      ],
+      '#description' => $this->t('Make the Global Banner sticky on top of the page.'),
```
Comment (22Alexandra) [inline modules/oe_webtools_globan/src/Form/WebtoolsGlobanSettingsForm.php:116]:
```suggestion
      '#description' => $this->t('Whether to make the banner sticky on top of the page or not.'),
```

File: modules/oe_webtools_globan/oe_webtools_globan.module [hunk 2]
```diff
@@ -61,6 +61,8 @@ function oe_webtools_globan_js_alter(&$javascript, AttachedAssetsInterface $asse
 
     $globan_param .= empty($config->get('display_eu_institutions_links')) ? '0' : '1';
 
+    $globan_param .= empty($config->get('sticky')) ? '0' : '1';
```
Comment (ademarco) [inline modules/oe_webtools_globan/oe_webtools_globan.module:64]: This hid the problem of not having an upgrade path under the carpet.

File: modules/oe_webtools_globan/oe_webtools_globan.post_update.php [hunk 3]
```diff
@@ -0,0 +1,22 @@
+<?php
+
+/**
+ * @file
+ * Post update hooks for Webtools Globan.
```
Comment (sergepavle) [inline modules/oe_webtools_globan/oe_webtools_globan.post_update.php:5]:
```suggestion
 * Post update hooks for Webtools Global Banner.
```

File: modules/oe_webtools_globan/oe_webtools_globan.post_update.php [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+ */
+
+declare(strict_types = 1);
+
+/**
+ * Set sticky config for globan.
+ */
+function oe_webtools_globan_post_update_00001(): void {
+  $config_factory = \Drupal::configFactory();
```
Comment (sergepavle) [inline modules/oe_webtools_globan/oe_webtools_globan.post_update.php:14]: Don't need to declare variable for once usage

Comment (sergepavle) [inline modules/oe_webtools_globan/oe_webtools_globan.post_update.php:14]: not necessarily declare variable here.
(optional comment)

File: composer.json [hunk 5]
```diff
@@ -6,7 +6,7 @@
     "minimum-stability": "dev",
     "prefer-stable": true,
     "require": {
-        "drupal/core": "^8.7.7",
+        "drupal/core": "^8.8",
```
Comment (sergepavle) [inline composer.json:9]: Are we finishing support of 8.7 officially?

File: modules/oe_webtools_globan/src/Form/WebtoolsGlobanSettingsForm.php [hunk 6]
```diff
@@ -106,6 +106,17 @@ public function buildForm(array $form, FormStateInterface $form_state): array {
...
 
+    $form['globan_settings']['sticky'] = [
+      '#type' => 'select',
+      '#title' => $this->t('Sticky'),
+      '#options' => [
+        0 => $this->t('No'),
+        1 => $this->t('Yes'),
+      ],
+      '#description' => $this->t('Whether to make the banner sticky on top of the page or not.'),
+      '#default_value' => empty($config->get('sticky')) ? 0 : 1,
```
Comment (sergepavle) [inline modules/oe_webtools_globan/src/Form/WebtoolsGlobanSettingsForm.php:117]: Maybe we have to also don't use anymore empty() https://github.com/openeuropa/oe_webtools/pull/124/files#diff-b05913734899aef67f064ef0c9ade8b1R64 for consistency

File: modules/oe_webtools_globan/src/Form/WebtoolsGlobanSettingsForm.php [hunk 7]
```diff
@@ -118,6 +129,7 @@ public function submitForm(array &$form, FormStateInterface $form_state): void {
       ->set('background_theme', $form_state->getValue('background_theme'))
       ->set('display_eu_institutions_links', (bool) $form_state->getValue('display_eu_institutions_links'))
       ->set('override_page_lang', $form_state->getValue('override_page_lang'))
+      ->set('sticky', $form_state->getValue('sticky'))
```
Comment (sergepavle) [inline modules/oe_webtools_globan/src/Form/WebtoolsGlobanSettingsForm.php:132]: It looks like missed casting to the boolean data type.
Config schema accepts boolean.

File: composer.json (hunk 5 above)
Comment (nagyad) [reply composer.json:9]: Yes, @ademarco proposed to do this. We are having issues with other dependencies that can't install on 8.7.

**Pull request merged**